tf2onnx/utils.py

```diff
@@ -121,17 +121,20 @@ def tf_to_onnx_tensor(tensor, name=""):
     new_type = TF_TO_ONNX_DTYPE[tensor.dtype]
     tdim = tensor.tensor_shape.dim
     dims = [d.size for d in tdim]
-    # FIXME: something is fishy here
-    if dims == [0]:
-        dims = [1]
     is_raw, data = get_tf_tensor_data(tensor)
+    # data is [], i.e., empty array
+    if dims == [0] and not is_raw and not data:
```

Comment:
use data ==[] instead of "not data"?

Reply:
agree, for empty tenosr, it should be```data is None```?

Reply:
we return None if we cannot get the tentor data. So here data is exactly None instead of "[]"

Reply:
okay, then use "data is None" for the check?